src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'f' ? 'husband' : 'wife';
+  const married = person.isMarried === true;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${person.name}`}
+      </h2>
+      {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+      )}
```

Comment:
don't forget about spaces
```suggestion
      {person.age && (
        <p className="Person__age">
          {`I am ${person.age}`}
        </p>
      )}
```